TheMealDB/TheMealDB/Moduls/CategroryScreen/Controller/CattegroryScreenController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CattegroryScreenController.swift
+//  TheMealDB
+//
+//  Created by mac on 12/18/18.
+//  Copyright © 2018 mac. All rights reserved.
+//
+
+import UIKit
+
+class CattegroryScreenController: UIViewController {
```

Comment:
```suggestion
class CategoryController: UIViewController {
```

viết sai chính tả nhiều quá